solidity/contracts/middleware/InterchainAccountRouter.sol

```diff
@@ -158,73 +166,6 @@ contract InterchainAccountRouter is Router, AbstractRoutingIsm {
     }
 
     // ============ External Functions ============
-    /**
-     * @notice Dispatches a single remote call to be made by an owner's
-     * interchain account on the destination domain
-     * @dev Uses the default router and ISM addresses for the destination
-     * domain, reverting if none have been configured
-     * @param _destination The remote domain of the chain to make calls on
-     * @param _to The address of the contract to call
-     * @param _value The value to include in the call
-     * @param _data The calldata
-     * @return The Hyperlane message ID
-     */
-    function callRemote(
```

Comment:
Do we feel like nobody is using this?

Reply:
I checked and I dont believe anyone is
this lets us use `calldata` everywhere